packages/gst-plugins-base/build.sh

```diff
@@ -19,6 +20,11 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 -Dexamples=disabled
 -Dpango=disabled
 "
+# This file is only relevant to Windows
+# and is not found in any gst-plugins package of other Linux distributions
+TERMUX_PKG_RM_AFTER_INSTALL="
```

Comment:
It would be better to check whether `include/GL/gl.h` exist in `post_massage` step.